### DISCUSSION TITLE
Bigger Data Partition

### DISCUSSION THREAD

**Discussion by RanchoHam** [procount/pinn#674] [Q&A]
I would like to increase the size of the data partition.  I could dink with the partition table after installation to sacrifice a project partition, but that offends my sense of software elegance.  Any ideas?

**Comment by procount** [comment]
1. You could use Matt's webpage: pinn.mjh.nz to choose your OSes and specify the partition sizes you want.
2. You could download the data partition installation files to a local USB stick (see README_PINN.md). Modify its partitions.json file to adjust its partition size to what you want. You could create several copies (with different os_names), each with different sizes to give you a greater choice.

**Reply by lurch** [reply]
3. You could download the data partition installation files to a local USB stick (see README_PINN.md). Modify its `partitions.json` file to set `"want_maximised": true,`
